frontend/types/enums.ts

```diff
@@ -5,6 +5,14 @@ export enum RoleCategory {
     ORGANIZATION = 'organization',
 }
 
+export enum ProjectSort {
+    MOST_STARS = 'most_stars',
+    MOST_DOWNLOADS = 'most_downloads',
+    MOST_VIEWS = 'most_views',
+    NEWEST = 'newest',
+    RECENTLY_UPDATED = 'recently_updated',
+}
+
```

Comment:
I would just have this as a local enum inside the component file. All the rest of the enums in this file have a mirrored enum in the java project, but this one doesn't. And like I said earlier, the ProjectSortingStrategy should be removed at some point. So I can't see this being used anywhere outside of that component.